code/modules/antagonists/heretic/heretic_antag.dm

```diff
@@ -1171,6 +1171,6 @@
 /datum/outfit/heretic
 	name = "Heretic (Preview only)"
 
-	suit = /obj/item/clothing/suit/hooded/cultrobes/eldritch
-	head = /obj/item/clothing/head/hooded/cult_hoodie/eldritch
+	suit = /obj/item/clothing/suit/hooded/cultrobes/eldritch/rust
+	head = /obj/item/clothing/head/hooded/cult_hoodie/eldritch/rust
```

Comment:
Just to double check, changing the heretic icon in antagonist selection was intentional on your part because they don't look like that any more?
If not, you should revert this rather than uploading a different screenshot.

Reply:
Plausibly in that case, shouldn't you give them a different weapon too? 😅 
I guess there isn't a "generic" version of the heretic blade other than that one.